sklearnex/dispatcher.py

```diff
@@ -73,9 +74,12 @@ def get_patch_map():
         from .neighbors import KNeighborsRegressor as KNeighborsRegressor_sklearnex
         from .neighbors import LocalOutlierFactor as LocalOutlierFactor_sklearnex
         from .neighbors import NearestNeighbors as NearestNeighbors_sklearnex
+        from .preview.cluster import KMeans as KMeans_sklearnex
```

Comment:
why is it moved?

Reply:
i guess it might be result of applying isort or black. i can't remember i did it. i'll fix it, thanks for mentioning